agents/hermes/plugin/__init__.py

```
@@ -1040,6 +1072,60 @@ def _active_managed_gateway_services():
+    if not target_match:
+        return response
+
+    message = _strip_wrapping_quotes(target_match.group("message"))
+    return message if message else response
+
+
+def _install_messaging_response_patch():
+    """Prevent raw messaging pseudo-tool calls from leaking as final text."""
+    try:
+        module = __import__("run_agent", fromlist=["AIAgent"])
+    except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)